autoload.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+defined( 'WPINC' ) || die;
+
+/**
+ * PSR-4 autoloader
+ */
+spl_autoload_register( function( $class )
+{
+	$namespaces = array(
+		'Mailgun_Subscriptions\\' => __DIR__ . '/Mailgun_Subscriptions/',
+	);
```

Comment:
Using a $namespaces array to allow multiple namespaces (i.e. for unit tests) in the future.